### ISSUE TITLE
Setup for parameters

### ISSUE THREAD

**Issue by Cli98** [rwightman/efficientdet-pytorch#25]
Hi @rwightman 

Just checked your code. Quite surprising to see your default training mode is to train without pre-train weight. How to train **with** pre-train weight? And normally, when we refer to pre-train weight, we are refer to pre-train weight for backbone right? So what's the point to present both "pretrain" and "pretrain backbone" option to user?

And what's drop_path rate and drop_block rate (did not see them in paper)? You set default value as None, so do you want your user to use them? If yes, can you provide default value?

I also want to know where is the yaml file in this line "args, args_text = _parse_args()"?

And **how to setup num_classes to correct number**?

Thanks for any help.

**Comment by rwightman** [comment]
You'll have to dig into these things yourself, the provided train script is currently focused on reproducing coco training results, you'll have to experiment and do a few modifications for your own.  Training from scratch is the hard part that no other impl get right. Improvements will be added in the future for fine-tuning.

See https://github.com/rwightman/efficientdet-pytorch#examples-of-training--fine-tuning-on-alternate-datasets to see how easy it is to modify bits of the training pipeline or reset the head for different classes....

**Comment by Cli98** [comment]
> You'll have to dig into these things yourself, the provided train script is currently focused on reproducing coco training results, you'll have to experiment and do a few modifications for your own. Training from scratch is the hard part that no other impl gets right. Improvements will be added in the future for fine-tuning.
> 
> See https://github.com/rwightman/efficientdet-pytorch#examples-of-training--fine-tuning-on-alternate-datasets to see how easy it is to modify bits of the training pipeline or reset the head for different classes....

So it is not possible to use your repo to train on customer dataset right?
